tests/sequences/__init__.py

```diff
@@ -0,0 +1 @@
+from .func import remove_duplicate_lists
```

Comment:
You're importing this in the wrong place.
The module you're looking for is `src/calc/__init__.py`.
This one should be fixed before merging.